dashdotdb/services/deploymentvalidation.py

```diff
@@ -161,10 +166,10 @@ def get_deploymentvalidation_summary():
         group by cluster.name
         """
 
-        token = db.session.query(
-            db.func.max(Token.id).label('token_id')
-        ).filter(
-            Token.id == DeploymentValidation.token_id,
+        latest = db.session.query(LatestTokens).filter(
+            data_type="DVOToken").first()
```

Comment:
nit: this data type constant should be defined somewhere generally. Even at a db schema level

Reply:
Not a nit, imo.  Magic strings, like magic numbers, are bad.  This is one of the reasons I labeled this WIP as I was still figuring out the best place to define this constant. :)

I've done it now so it'll be in the updated changes.